enterprise/utils.py

```diff
@@ -533,13 +606,34 @@ def enterprise_course_enrollment_model():
     return apps.get_model('enterprise', 'EnterpriseCourseEnrollment')  # pylint: disable=invalid-name
 
 
+def enterprise_bulk_enroll_template_model():
+    """
+    Returns the ``BulkEnrollmentNotificationEmailTemplate`` class.
+    """
+    return apps.get_model('enterprise', 'BulkEnrollmentNotificationEmailTemplate')  # pylint: disable=invalid-name
+
+
 def licensed_enterprise_course_enrollment_model():  # pylint: disable=invalid-name
     """
     returns the ``LicensedEnterpriseCourseEnrollment`` class.
     """
     return apps.get_model('enterprise', 'LicensedEnterpriseCourseEnrollment')  # pylint: disable=invalid-name
 
 
+def get_enterprise_bulk_enroll_email_template():
+    """
+    Get the first instance found of `BulkEnrollmentNotificationEmailTemplate`
+    Currently we only expect to use one of these, since it's not per customer.
```

Comment:
What is the future of this? Is there a future of this or do we really only ever expect 1? Because if we only ever expect 1, I'm extra not a fan of a new model.
We should honestly design this for the future unless that future is 100% "we're just gonna stick it in Braze"

Reply:
I think the future is going to be we use Braze.. since it looks like we are standardizing on it. So this I see as a stop gap only. but who knows how long this will stay here? I also confirmed with Joe/Jen that we are not looking for per customer customization of this template at this time.